dap.md

```diff
@@ -1,79 +0,0 @@
-## Hosting the DAP Snippet
```

Comment:
I removed this outdated, never-used markdown document I once wrote that describes how to build and package the DAP JS in a process that was never adopted by DAP.